### PR TITLE
Create a database based on a diagram

### PR DESCRIPTION
**Pull request by carreraprogrammer**
## HOSPITAL DATABASE

In this project, I created a Database, based on a schema

clinic_diagram

- [x] Create a file named schema_based_on_diagram.sql where you implement the database from the diagram.

- [x] Join tables from many-to-many relationships might not appear in the diagram, but you still need them.

- [x] Remember to add the FK indexes.

Captura desde 2023-06-14 15-25-57

### CODE REVIEW

File: schema_based_on_diagram.sql [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+CREATE TABLE invoice_items (
+id int PRIMARY KEY IDENTITY (1,1),
+unit_price decimal,
+quantity int,
+total_price decimal,
+invoice_id int,
+treatment_id int,
+FOREIGN KEY (treatment_id) REFERENCES treatments(id),
+FOREIGN KEY (invoice_id) REFERENCES invoices(id)
+);
```
Comment (Olamarx) [inline schema_based_on_diagram.sql]: - [x] I think this is a duplicate, kindly remove it.

**Review by Olamarx — CHANGES_REQUESTED** [review]
Hi @carreraprogrammer, 

Good job so far, indeed you have spent time on this project!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
- Added Foreign key indexes ✔️
- Create join table ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema_based_on_diagram.sql (hunk 1 above)
Comment (carreraprogrammer) [reply schema_based_on_diagram.sql]: Hello @Olamarx. Thanks for the review, I did the changes 😁

**Review by Reem-lab — APPROVED** [review]
# Hi @carreraprogrammer,  👋🏻👋🏻

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights
- [x] Great job on the funtionality 💯
- [x] Followed Gitflow 🔥
- [x] All requested changes are applied⭐


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by carreraprogrammer** [comment]
Hello @Reem-lab, thanks for the review 😁🎊🎉

**Pull request merged**